tests/testthat/test-rng.R

```diff
@@ -1178,3 +1178,75 @@ test_that("numbers on different streams behave as expected", {
   expect_equal(res[, 2],
                dust_rng$new(1, seed = 1)$jump()$hypergeometric(10, m, n, k))
 })
+
+
+test_that("gamma for a = 1 is the same as exponential", {
+  rng1 <- dust_rng$new(seed = 1L)
+  rng2 <- dust_rng$new(seed = 1L)
+
+  n <- 10
+  b <- 3
+
+  gamma <- rng1$gamma(n, 1, b)
+  exp <- rng2$exponential(n, 1 / b)
+
+  expect_equal(gamma, exp)
+})
+
+
+test_that("can draw gamma random numbers", {
+  ## a > 1
```

Comment:
```suggestion
  ## when a > 1
```

avoid lint false positive (and below)